app/models/aspace_version_control/get_eads_job.rb

```diff
@@ -10,7 +10,7 @@ class GetEadsJob < LibJob
     def initialize(local_git_lab_dir: Rails.application.config.aspace.local_git_lab_dir)
       super(category: "EAD_export")
       @errors = []
-      @local_git_lab_dir = local_git_lab_dir
+      @local_git_lab_dir = ENV['GIT_LAB_DIR'] || local_git_lab_dir
```

Comment:
One question (not a blocker) -- to me, having an environment variable that can be set to any directory on the machine seems to imply that any directory is valid.  But there are in reality only two valid directories (the gitlab staging and prod directories).  I wonder if it would be worth adding some kind of guard in case the user tries to run the job with an invalid GIT_LAB_DIR envvar?